asv_benchmarking/asv.conf.json

```diff
@@ -0,0 +1,73 @@
+{
+    // The version of the config file format.  Do not change, unless
+    // you know what you are doing.
+    "version": 1,
+
+    // The name of the project being benchmarked
+    "project": "lmfit-py",
+
+    // The project's homepage
+    "project_url": "http://lmfit.github.io/lmfit-py",
+
+    // The URL or local path of the source code repository for the
+    // project being benchmarked
+    "repo": "http://github.com/lmfit/lmfit-py/",
+
+    // List of branches to benchmark. If not provided, defaults to "master"
+    // (for git) or "tip" (for mercurial).
+    "branches": ["master", "param_minmax_inf"], // for git
+    // "branches": ["tip"],    // for mercurial
```

Comment:
I don't think we'd want the `param_minmax_inf` branch hardcoded in there. Can this just be the master branch, and whatever branch is currently being worked on?